examples/itowns_GLB.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Display a Glb/Gltf object with iTowns</title>
+
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" type="text/css" href="css/example.css" />
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="../dist/itowns.js"></script>
+        <script type="text/javascript">
+            var THREE = itowns.THREE
+            // iTowns namespace defined here
+            var viewerDiv = document.getElementById("viewerDiv");
+
+            var placement = {
+                coord: new itowns.Coordinates(
+                    "EPSG:4326",
+                    4.827361189714537,
+                    45.78537982365657,
+                    10
+                ),
+                tilt: 22,
+                heading: 0,
+                range: 2840,
+            };
+
+            var view = new itowns.GlobeView(viewerDiv, placement);
+
+            var orthoSource = new itowns.TMSSource({
+                crs: "EPSG:3857",
+                isInverted: true,
+                format: "image/png",
+                url: "http://osm.oslandia.io/styles/klokantech-basic/${z}/${x}/${y}.png",
+                attribution: {
+                    name: "OpenStreetMap",
+                    url: "http://www.openstreetmap.org/",
+                },
+                tileMatrixSet: "PM",
+            });
+
+            var orthoLayer = new itowns.ColorLayer("Ortho", {
+                source: orthoSource,
+            });
+
+            view.addLayer(orthoLayer);
+
+            // Load a new gltf object
+            itowns.glTFLoader.load(
+                "https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/lampadaire/scene.gltf ",
+                (gltf) => {
+                    var model = gltf.scene;
+                    model.scale.set(10,10,10)
+                    model.updateMatrixWorld();
+                    model.rotation.set(Math.PI / 2.1, 0, -Math.PI / 4.1);
```

Comment:
Why `Math.PI / 2.1` and not `Math.PI / 2.0`? (same for `4.1`)

Reply:
I think you need to apply the following two rotations to correctly align yout model with the ground:
* `Math.PI / 2.0` to X axis to go from gltf y-up convention to itowns z-up convention
* One rotation to align the z-up axis of your model with the geodesic normal of the globe where your model is positioned. You can use [Coordinates.geodesicNormal](https://github.com/iTowns/itowns/blob/master/src/Core/Geographic/Coordinates.js#L191) to get it. We do something similar in the [collada example](https://github.com/iTowns/itowns/blob/master/examples/misc_collada.html#L100)

Reply:
I didn't really understand the whole thing but I made something better I hope.